checks/windows.py

```diff
@@ -350,3 +350,37 @@ def checkWindowsARM64EmulationStatus(lines):
     if (len(search('Windows ARM64: Running with x64 emulation', lines)) > 0):
         return [LEVEL_WARNING, "x64 OBS on Windows ARM64",
                 "You are running the x64 version of OBS on an ARM64 Windows system. This will greatly reduce performance due to the mandatory use of emulation."]
+
+
+def getDshowLines(lines):
+    dshowPos = []
+    for i, s in enumerate(lines):
+        if 'DShow Device:' in s:
+            dshowPos.append(i)
+    return dshowPos
+
+
+def checkDshowInterference(lines):
+    if (getDshowLines(lines)):
+        dshowLines = getDshowLines(lines)
+        dshowDevices = []
+        videoPaths = []
+
+        for i in dshowLines:
+            fixedPath = lines[i + 2][:-1]
+            fixedPath = fixedPath.split('\t', 1)[1]
+            dshowDevices.append(lines[i][:-1])
+            videoPaths.append(fixedPath)
+
+        pathList = []
+        duplicateList = []
+        for i in videoPaths:
+            if i not in pathList:
+                pathList.append(i)
+            else:
+                if (i != 'video path: '):
+                    duplicateList.append(i)
+
+        if (duplicateList):
+            return [LEVEL_WARNING, "Video Capture Device Interference",
+                    "You have multiple Video Capture Device sources pointed to the same device. This is known to cause issues such as Video Capture Device sources working intermittently or not working at all. Remove any duplicates and use the <strong>Add Existing</strong> option when adding sources to new scenes."]
```

Comment:
Is this language pulled from the support bot or is it brand new?

Reply:
It is new, currently we have no checks or bot commands related to this.